control-plane-operator/controllers/hostedcontrolplane/etcd/params.go

```diff
@@ -10,60 +12,65 @@ import (
 )
 
 type EtcdParams struct {
-	ClusterVersion            string
-	EtcdOperatorImage         string
-	OwnerRef                  config.OwnerRef `json:"ownerRef"`
-	OperatorDeploymentConfig  config.DeploymentConfig
-	EtcdDeploymentConfig      config.DeploymentConfig
-	PersistentVolumeClaimSpec *corev1.PersistentVolumeClaimSpec `json:"persistentVolumeClaimSpec"`
+	EtcdOperatorImage string
+
+	OwnerRef         config.OwnerRef `json:"ownerRef"`
+	DeploymentConfig config.DeploymentConfig
+
+	SnapshotInterval time.Duration
```

Comment:
Maybe add a comment here that this doesn't actually do anything because it uses the noop provider?